src/EFCore.PG/Design/Internal/NpgsqlCSharpRuntimeAnnotationCodeGenerator.cs

```diff
@@ -82,15 +82,8 @@ public class NpgsqlCSharpRuntimeAnnotationCodeGenerator : RelationalCSharpRuntim
 
         switch (typeMapping)
         {
-#pragma warning disable CS0618 // NpgsqlConnection.GlobalTypeMapper is obsolete
             case NpgsqlEnumTypeMapping enumTypeMapping:
-                if (enumTypeMapping.NameTranslator != NpgsqlConnection.GlobalTypeMapper.DefaultNameTranslator)
-                {
-                    throw new NotSupportedException(
-                        "Mapped enums are only supported in the compiled model if they use the default name translator");
-                }
-                break;
-#pragma warning restore CS0618
+                throw new NotImplementedException("Need to bake the enum values into the compiled model");
```

Comment:
TODO: NpgsqlEnumTypeMapping no longer references the name translator, but rather the mapping table from enums to their PG labels. This is better in general but I needed to serialize that properly into the compiled model.